lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -220,6 +221,71 @@ def test_enroll_nouser_change_autoenroll(self):
 
         return self._run_state_change_test(before_ideal, after_ideal, action)
 
+    @ddt.data(True, False)
+    def test_enroll_inactive_user(self, auto_enroll):
+        before_ideal = SettableEnrollmentState(
+            user=True,
+            enrollment=False,
+            allowed=False,
+            auto_enroll=False,
+        )
+        print("checking initialization...")
```

Comment:
Remove print functions?

Reply:
@bradenmacdonald, this is probably a side effect of copy-pasting code from an existing test in the same file (there are a few not including the changes in this PR) and adapting it to the thing that I wanted to test. Should I also remove the other, unrelated `print` functions as well?

Reply:
I guess if they're already there and they're suppressed by the test runner unless the test fails, they're not doing any harm. I just don't usually see print()s left in code.